### PR TITLE
Adding tests for clickhouse-keeper add test_048 into test_operator.py and adopt test_keeper.py to separate type

### PR DESCRIPTION
**Pull request by antip00**
Adding test regarding adding support for clickhouse-keeper
https://github.com/Altinity/clickhouse-operator/pull/1218

### CODE REVIEW

File: docs/chk-examples/clickhouse-keeper-1-node-for-test-only.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+metadata:
+  # DNS would be like zookeeper.namespace.svc
+  name: zookeeper
+  labels:
+    app: zookeeper
+spec:
+  ports:
+    - port: 2181
+      name: client
+    - port: 7000
```
Comment (Slach) [inline docs/chk-examples/clickhouse-keeper-1-node-for-test-only.yaml:14]: keeper is not provided different port for prometheus 7000, remote it

File: docs/chk-examples/clickhouse-keeper-1-node-for-test-only.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    what: node
+---
+apiVersion: "clickhouse-keeper.altinity.com/v1"
+kind: "ClickHouseKeeperInstallation"
+metadata:
+  name: clickhouse-keeper
+spec:
+  configuration:
+    clusters:
+      - name: "simple-1"
```
Comment (Slach) [inline docs/chk-examples/clickhouse-keeper-1-node-for-test-only.yaml:27]: you need to check keeper 3 -> 1 -> 3 -> 1 scenario, look to already implements `test_keeper.py`

File: tests/e2e/util.py [hunk 3]
```diff
@@ -82,9 +82,9 @@ def require_keeper(keeper_manifest="", keeper_type="zookeeper", force_install=Fa
             keeper_manifest = f"../../deploy/zookeeper/zookeeper-manually/quick-start-persistent-volume/{keeper_manifest}"
         if keeper_type == "clickhouse-keeper":
             keeper_manifest = (
-                "clickhouse-keeper-1-node-256M-for-test-only.yaml" if keeper_manifest == "" else keeper_manifest
+                "clickhouse-keeper-1-node-for-test-only.yaml" if keeper_manifest == "" else keeper_manifest
             )
-            keeper_manifest = f"../../deploy/clickhouse-keeper/clickhouse-keeper-manually/{keeper_manifest}"
+            keeper_manifest = f"../../deploy/clickhouse-keeper/clickhouse-keeper-with-CHK-resource/{keeper_manifest}"
```
Comment (Slach) [inline tests/e2e/util.py:87]: it will broke `test_keeper.py`

File: tests/e2e/util.py [hunk 4]
```diff
@@ -98,7 +98,7 @@ def require_keeper(keeper_manifest="", keeper_type="zookeeper", force_install=Fa
                 keeper_nodes = doc["spec"]["replicas"]
         expected_docs = {
             "zookeeper": 5 if "scaleout-pvc" in keeper_manifest else 4,
-            "clickhouse-keeper": 7,
+            "clickhouse-keeper": 2,
```
Comment (Slach) [inline tests/e2e/util.py:101]: it will broke `test_keeper.py`

**Review by Slach — CHANGES_REQUESTED** [review]

File: tests/e2e/kubectl.py [hunk 5]
```diff
@@ -138,8 +138,10 @@ def create_and_check(manifest, check, ns=None, shell=None, timeout=1800):
     else:
         # Wait for reconcile to start before performing other checks. In some cases it does not start, so we can pass
         # wait_field_changed("chi", chi_name, state_field, prev_state, ns)
-        wait_chi_status(chi_name, "InProgress", ns=ns, retries=3, throw_error=False, shell=shell)
-        wait_chi_status(chi_name, "Completed", ns=ns, shell=shell)
+        # wait_field(kind=kind, name=chi_name, field=".status.status", value="InProgress"
+        #            , ns=ns, retries=3, throw_error=False, shell=shell)
```
Comment (Slach) [inline tests/e2e/kubectl.py:142]: Why do you remove `InProgress`?

it covers corner cases when nothing happens and chi just not reconciled and not changed

**Review by Slach — CHANGES_REQUESTED** [review]

File: docs/chk-examples/clickhouse-keeper-1-node-for-test-only.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        layout:
+          replicasCount: 1
```
Comment (Slach) [inline docs/chk-examples/clickhouse-keeper-1-node-for-test-only.yaml:29]: let's change it to replicasCount: 3

**Review by Slach — CHANGES_REQUESTED** [review]

File: docs/chk-examples/clickhouse-keeper-3-node-for-test-only.yaml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+---
+# Fake Service to drop-in replacement in tests
+apiVersion: v1
```
Comment (Slach) [inline docs/chk-examples/clickhouse-keeper-3-node-for-test-only.yaml:3]: let's use this manifest in test_048

**Review by Slach — APPROVED** [review]
